src/modules/sync/diff/assetFolder.ts

```diff
@@ -0,0 +1,12 @@
+import { AssetFolderSyncModel } from "../types/syncModel.js";
+import { baseHandler, Handler, makeArrayHandler, makeObjectHandler } from "./combinators.js";
+
+export const assetFolderHandler: Handler<AssetFolderSyncModel> = makeObjectHandler({
+  name: baseHandler,
+  folders: { contextfulHandler: () => makeArrayHandler(f => f.codename, assetFolderHandler) },
+});
+
+export const assetFoldersHandler: Handler<ReadonlyArray<AssetFolderSyncModel>> = makeArrayHandler(
+  f => f.codename,
+  assetFolderHandler,
```

Comment:
we need to reorder these operations because of the workaround for `move` operation (remove an add). Remove needs to be first, otherwise it will throw error. 

It's time for 

> Comparator 🥳🥳

Reply:
I had to split it into two requests, because the MAPI refused requests where I first removed a folder and then added a folder with the same codename...